From c35f78d7a422750917029d20d9e57000b1181d75 Mon Sep 17 00:00:00 2001
From: Alexey Kardashevskiy <aik@ozlabs.ru>
Date: Mon, 14 Jan 2019 11:40:27 +1100
Subject: [PATCH] powerpc/powernv: Remove never used pnv_power9_force_smt4

References: bsc#1061840
Patch-mainline: v5.1-rc1
Git-commit: c35f78d7a422750917029d20d9e57000b1181d75

This removes never used symbol - pnv_power9_force_smt4.

Note that we might still want to add stubs for:
	void pnv_power9_force_smt4_catch(void);
	void pnv_power9_force_smt4_release(void);

Fixes: 7672691a08c88 "powerpc/powernv: Provide a way to force a core into SMT4 mode"
Signed-off-by: Alexey Kardashevskiy <aik@ozlabs.ru>
Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
Acked-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/include/asm/powernv.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/powerpc/include/asm/powernv.h b/arch/powerpc/include/asm/powernv.h
index 2f3ff7a27881..362ea12a4501 100644
--- a/arch/powerpc/include/asm/powernv.h
+++ b/arch/powerpc/include/asm/powernv.h
@@ -40,7 +40,6 @@ static inline int pnv_npu2_handle_fault(struct npu_context *context,
 }
 
 static inline void pnv_tm_init(void) { }
-static inline void pnv_power9_force_smt4(void) { }
 #endif
 
 #endif /* _ASM_POWERNV_H */
-- 
2.20.1

